dietpi/dietpi-software

```diff
@@ -5803,6 +5811,29 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 
 		fi
 
+		software_id=192 # K3s
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			INSTALL_URL_ADDRESS='https://get.k3s.io'
+			G_CHECK_URL "$INSTALL_URL_ADDRESS"
+
+      # Fetch config file if it exists
+			if [[ -f '/boot/dietpi-k3s.yaml' ]]; then
+				G_EXEC mkdir -p '/etc/rancher/k3s/'
+				G_EXEC cp '/boot/dietpi-k3s.yaml' '/etc/rancher/k3s/config.yaml'
+			fi
+
+			# Install
+			G_EXEC curl -sSfL "$INSTALL_URL_ADDRESS" -o install.sh
```

Comment:
Why not use `Download_Install` here?
```
Download_Install 'https://get.k3s.io' install.sh
```

Reply:
Overkill, I'd say. For single files with a known fixed URL address, Download_Install has no advantage.